app/insomnia/scratch-pad.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Local storage 
+
+content_type: concept
+layout: concept
+
+products:
+    - insomnia
+
+related_resources:
+  - text: Storage options
+    url: /insomnia/storage-options
+  - text: Git sync
+    url: /insomnia/git-sync
+  - text: Cloud sync
+    url: /insomnia/cloud-sync
+  - text: API specs
+    url: /insomnia/api-specs/
+  - text: Write tests for HTTP status codes
+    url: /how-to/write-http-status-tests/
+  - text: Security at Insomnia
+    url: /
+
+---
+
+Local vault (scratch pad) is a storage option that allows all project data to be stored locally on your device. This option is ideal for users who prefer or require their data to remain off the cloud for privacy or security reasons.
```

Comment:
Local vault and scratch pad aren't the same thing, I would has separate pages for them

Reply:
Thanks! I didn't realize it